### PR TITLE
QA Observation: Bugfix/FOUR-12502 The fields are not cleared after closing the modal

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
- Modal window clear all fields on Close

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-836ac54264.eng.processmaker.net

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by ana-mauricio — CHANGES_REQUESTED** [review]
En esta parte la seleccion de usuario est oculta. SI se cambia el usuario por otro este se queda por defecto y los siguientes scripts seran creados con ese usuario.

El campo user, no se limpia despues que el modal se cierra como se muestra en el video:

https://github.com/ProcessMaker/processmaker/assets/21198450/a693f54a-14c8-4ae9-9f25-a66e647aeee1

@fagubla @pmPaulis

File: resources/js/processes/scripts/components/CreateScriptModal.vue [hunk 1]
```diff
@@ -17,6 +17,7 @@
       no-close-on-backdrop
       :ok-disabled="disabled"
       :title="modalSetUp"
+      @hidden="modalHidden"
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/processes/scripts/components/CreateScriptModal.vue]: Solved

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-836ac54264.eng.processmaker.net

**Review by ana-mauricio — APPROVED** [review]

**Review by fagubla — APPROVED** [review]

**Pull request merged**